flink-kubernetes-webhook/pom.xml

```diff
@@ -36,6 +36,19 @@ under the License.
             <groupId>org.apache.flink</groupId>
             <artifactId>flink-kubernetes-operator</artifactId>
             <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>io.javaoperatorsdk</groupId>
+            <artifactId>operator-framework-framework-core</artifactId>
+            <version>${operator.sdk.admission-controller.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
```

Comment:
The admission-controller-framework is depending on the `io.fabric8:openshift-client` that pulled in an older `io.fabric8:kubernetes-client`.  It had a collision with the newer `kubernetes-client` version that we put on the class path with the the operator-shaded jar.

Reply:
I think the slightly cleaner approach here would be to remove the exclude all and explicitly include artifacts in the shade plugin:

```
<configuration>
	<artifactSet>
		<includes>
			<include>io.javaoperatorsdk:operator-framework-framework-core</include>
		</includes>
	</artifactSet>
	<transformers combine.children="append">
            <transformer
                    implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
                <mainClass>org.apache.flink.kubernetes.operator.admission.FlinkOperatorWebhook</mainClass>
            </transformer>
        </transformers>
</configuration>
``` 

This will be more flexible in the future when we need to add 1-2 extra packages here

Reply:
Thanks @gyfora for the suggestion, I like it better myself. Changed and pushed it.

Reply:
Had to keep the excludes in the dependency for the unit tests.